install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+cd /vagrant
+
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+curl -L https://get.rvm.io | bash -s stable
+
+source $HOME/.rvm/scripts/rvm
+rvm use --install 2.2
+
+bundle update
```

Comment:
I don't think this should be part of the setup process - as I understand, `bundle update` will install the most recent version of all the gems, versus what is pinned in the `Gemfile.lock`